lib/DevTools.js

```diff
@@ -0,0 +1,54 @@
+import {connectViaExtension, extractState} from 'remotedev';
```

Comment:
Hey @pac-guerreiro I'm really curious on how you decided on using `remotedev`! I'm relatively new to  React/ReactNative so would love to know : D

I could only find [this slack post](https://expensify.slack.com/archives/C01GTK53T8Q/p1690385918332959) you sent recently on this.

(btw, this is super cool)

Reply:
@PauloGasparSv 

Sorry for the late response, I'm glad you like this idea 😄 I hope it helps a lot of developers working on the app 😄 

Well, at first I thought I could fork the repo for redux dev tools extension but then I found it was too complex to adapt it as it was for `Onyx`, so after some digging and research I found `remotedev`, which was made by the same guy behind redux dev tools extension, if I'm not mistaken. It basically provides an api to connect to redux dev tools or any other extension that supports `remotedev`, so I gave it a shot and it worked 😄